sensor_msgs/Image depth
sensor_msgs/Image rgb
sensor_msgs/Image init_mask
---
sensor_msgs/Image mask
float32[] plane 
